lib/ui/contact_list/group_chat_details_sheet.dart

```diff
@@ -117,13 +92,25 @@ class _GroupChatDetailsSheetState extends ConsumerState<GroupChatDetailsSheet> w
       final contactsWithKeyPackage = filteredContacts['withKeyPackage']!;
       final contactsWithoutKeyPackage = filteredContacts['withoutKeyPackage']!;
 
-      if (contactsWithKeyPackage.isEmpty) {
-        safeShowErrorToast('No contacts have keypackages available for group creation');
+      // If less than 2 contacts have keypackages, only show invite sheet (no group creation)
```

Comment:
Ok, s i don't want to be a pain in the ass, but I still think that this is unexpected for the user...  I could want to create a group of 2 people, I think it might be a cool feature. I can do that in signal. I tried to create a group now and sheet gets closed without feedback of an error. Don't you think that it would be better to remove that condition? And just leave the if below of `contactsWithoutKeyPackage.isNotEmpty && mounted`